lib/Conversion/TorchToLinalg/Reduction.cpp

```diff
@@ -535,6 +547,9 @@ class ConvertReductionOp : public ConversionPattern {
         !elemType.isa<mlir::FloatType>())
       return rewriter.notifyMatchFailure(
           op, "only float types are valid for vector norm ops");
+    if((isa<AtenAllDimOp>(op)) && elemType.isa<mlir::IntegerType>() && 
```

Comment:
unnecessary outer braces (and formatting looks weird to me)

Reply:
Missed that, fixed